config/production.js

```diff
@@ -15,8 +15,15 @@ module.exports = {
     accountKey: ENV('AZURE_ACCOUNT_KEY')
   },
   githubConfig: {
+    username: ENV('GITHUB_USERNAME'),
     token: ENV('GITHUB_TOKEN')
   },
+  // A test user and token which does not have access to the autolander-test repo.
+  // This is only used for testing support and does not need to be defined for production.
+  githubTestUser: {
```

Comment:
This is not necessary to be defined in this file, but I just do so in order to keep the configs roughly the same for each.